src/app/cras/edit-cra/edit-cra.component.spec.ts

```diff
@@ -0,0 +1,25 @@
+import { async, ComponentFixture, TestBed } from '@angular/core/testing';
+
+import { EditCraComponent } from './edit-cra.component';
+
+describe('EditCraComponent', () => {
+  let component: EditCraComponent;
+  let fixture: ComponentFixture<EditCraComponent>;
+
+  beforeEach(async(() => {
+    TestBed.configureTestingModule({
+      declarations: [ EditCraComponent ]
+    })
+    .compileComponents();
+  }));
+
+  beforeEach(() => {
+    fixture = TestBed.createComponent(EditCraComponent);
+    component = fixture.componentInstance;
+    fixture.detectChanges();
+  });
+
+  it('should create', () => {
```

Comment:
You have some test to add here, valid the `init`, `onSubmitCra` and other method.
Validate if the `edit` / `consulting` mode works correctly should be nice too